CI/flatpak/com.obsproject.Studio.json

```diff
@@ -21,6 +21,9 @@
     "--system-talk-name=org.freedesktop.Avahi",
     "--env=VST_PATH=/app/extensions/Plugins/lxvst"
   ],
+  "inherit-extensions": [
+    "org.freedesktop.Platform.VAAPI.Intel"
```

Comment:
This looks really unexpected. The extension point is declared in runtime and should just work.

Reply:
Having `LIBVA_DRIVER_NAME` transferred to the Flatpak env was also unexpected for me.